File: src/PBR.Application/Models/AccountModel.cs
using PBR.Application.Models.Base;
using System;
using System.Collections.Generic;
using System.Text;

namespace PBR.Application.Models
{
    public class AccountModel:BaseModel
    {
        public int Id { get; set; }
        public string UserName { get; set; }
        public string Passoword { get; set; }
        public string ClientId { get; set; }
        public string ClientSecret { get; set; }
        public string AccessToken { get; set; }
        public string RefreshToken { get; set; }
        public int ExpiresIn { get; set; }
        public int ExpiresOn { get; set; }
        public bool IsActive { get; set; }
        public DateTime CreatedDate { get; set; }
        public DateTime LastUpdatedDate { get; set; }
        public string AccountName { get; set; }
        public bool IsDelete { get; set; }
    }
}

File: src/PBR.UI/src/app/pages/admin/account/account.model.ts
import { DatePipe, formatDate } from '@angular/common';
import { Inject, LOCALE_ID } from '@angular/core';

export class Accounts {
  id: number;
  userName: string;
  password: string;
  clientId: string;
  clientSecret: string;
  accessToken: string;
  refreshToken: string;
  expiresIn: number;
  expiresOn: number;
  isActive: false;
  createdDate: string;
  lastUpdatedDate: string;
  accountName: string;
  accountList: any;

  /**
   *
   */
  constructor() {}

  fromJson(data: Accounts[]) {
    const accounts: any[] = [];

    data.forEach((acc) => {
      accounts.push({
        id: acc.id,
        userName: acc.userName,
        password: acc.password,
        clientId: acc.clientId,
        isActive: acc.isActive,
        clientSecret:acc.clientSecret,
        lastUpdatedDate: formatDate(
          acc.lastUpdatedDate,
          'dd/MM/yyyy HH:mm:ss',
          'en-US'
        ),
        accountName: acc.accountName,
      });
    });

    return accounts;
  }

  toJson() {}
}
